spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -506,3 +506,65 @@ If `end()` is called, and the current conversation is marked transient, an `Ille
 If `begin()` is called, and the current conversation is already marked long-running, an `IllegalStateException` is thrown.
 
 If `begin()` is called with an explicit conversation identifier, and a long-running conversation with that identifier already exists, an `IllegalArgumentException` is thrown.
+
+=== Context control for built-in scopes
+
+The container provides support for automatically starting contexts based on occurrences within the container.  There are cases where manually starting and stopping a context
```

Comment:
I'm not sure, maybe something like _"The container usually activates and destroys the built-in contexts automatically. However, there are cases where manually starting..."_